x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/parser/ParserUtils.java

```diff
@@ -25,6 +25,11 @@
 import static java.util.Collections.emptyList;
 
 public final class ParserUtils {
+    public enum ParamClassification {
```

Comment:
This looks like a duplicate of ParamParsingKeys.

Reply:
They do look very similar, after using enum kind in params, it is less confusing.